### PR TITLE
Show new paper keys in the devices tab after creation

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

We weren't refreshing the devices list after creating a new paper key.

### CODE REVIEW

File: shared/devices/gen-paper-key/index.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ type State = {
 type Props = {
   paperKey: HiddenString,
   generatePaperKey: () => void,
-  onBack: () => void
+  onBack: () => void,
+  onFinish: () => void
```
Comment (chromakode) [inline shared/devices/gen-paper-key/index.js]: Trailing comma here?

**Comment by cjb** [comment]
@chromakode I don't use trailing commas in flow types:

Most of our flow types don't have trailing commas, because they predate our trailing commas style and were unincluded in the eslint --fix because eslint doesn't grok flow. Nojima argues we shouldn't have a style we can't enforce, so I leave them out to stay consistent.

What do you think?

**Comment by chromakode** [comment]
@cjb The code I've written and seen in review is putting trailing commas everywhere. I think that it makes sense to use a consistent comma style for new code we write, but not worth spending much time on though since we can't lint it. I wouldn't make a point of \*not\* using trailing commas to match old code though -- we prefer trailing commas because they make the code a little bit easier to change with in the future. That comment was definitely a :sheep:, just something I noticed while perusing the rest of the code.

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**